pkg/keys/ed25519.go

```diff
@@ -83,7 +95,7 @@ func GenerateEd25519Key() (*ed25519Signer, error) {
 	}, nil
 }
 
-func NewEd25519Signer(keyValue Ed25519PrivateKeyValue) *ed25519Signer {
+func NewEd25519SignerFromKey(keyValue Ed25519PrivateKeyValue) *ed25519Signer {
```

Comment:
This is related to - https://github.com/theupdateframework/go-tuf/pull/275/files/d5af22b783122f7c7ad81ca9b1b8725cdadad35e#diff-995becca79d6625fcb4d962991fa15f34c77a03a2dd760e810c31410a22a3300R22. I have to reuse the prototype to allow the builder to be added to the registry (SignerMap). And due to the fact that Go doesn't support prototype overrides, I had to create a new function.

Furthermore, it will break by design because a caller who was using `NewP256Signer` to instantiate a signer was in fact instantiating an `ed25519` one.

Reply:
Sounds good. Let's be sure to communicate this change in our next release